docs/timeseries.md

```diff
@@ -10,7 +14,7 @@ Accept headers: `application/json`, `text/csv`, `application/geo+json`
 
```

Comment:
Add a single line here with a brief explanation of how to use an Accept Header, something like:

Accept headers are a way to have more control over the output that is returned from Hydrocron.

Reply:
Added: 

Accept headers provide more control over the output that is returned by Hydrocron. You can pass the Accept header in your request to return a specific response format.